domain/src/main/java/uk/gov/hmcts/ccd/definition/store/domain/validation/displaycontextparameter/DateTimeEntryValidatorImpl.java

```diff
@@ -8,10 +8,13 @@
 
 import java.util.regex.Pattern;
 
+import static uk.gov.hmcts.ccd.definition.store.repository.FieldTypeUtils.BASE_DATE;
+
 @Component
 public class DateTimeEntryValidatorImpl implements DisplayContextParameterValidator {
 
-    private static final Pattern ALLOWED_CHARACTERS_PATTERN = Pattern.compile("[yMdHmsS\\-'T:]+");
+    private static final Pattern NOT_ALLOWED_CHARACTERS_PATTERN_DATETIME = Pattern.compile("VzOXxZ");
```

Comment:
can we move these constants to the interface to re-use in implementation classes?